Library/Homebrew/brew.sh

```diff
@@ -159,9 +184,9 @@ else
 
   # Ensure the system Curl is a version that supports modern HTTPS certificates.
   HOMEBREW_MINIMUM_CURL_VERSION="7.41.0"
-  system_curl_version_output="$($(command -v curl) --version 2>/dev/null)"
-  system_curl_name_and_version="${system_curl_version_output%% (*}"
-  if [[ $(numeric "${system_curl_name_and_version##* }") -lt $(numeric "$HOMEBREW_MINIMUM_CURL_VERSION") ]]
+  curl_version_output="$($HOMEBREW_CURL --version 2>/dev/null)"
+  curl_name_and_version="${curl_version_output%% (*}"
+  if [[ $(numeric "${curl_name_and_version##* }") -lt $(numeric "$HOMEBREW_MINIMUM_CURL_VERSION") ]]
   then
     HOMEBREW_SYSTEM_CURL_TOO_OLD="1"
```

Comment:
This environment variable is no longer correct if it's not checking the system version

Reply:
Would you recommend renaming it to `HOMEBREW_CURL_TOO_OLD` and using similarly to `HOMEMEBREW_SYSTEM_CURL_TOO_OLD`?

Reply:
I see two possible scenarios:

1. Introduce `HOMEBREW_CURL_TOO_OLD` and use it the same way `HOMEBREW_SYSTEM_CURL_TOO_OLD` is used.

2. Emit an error message if `HOMEBREW_CURL` is set but is too old.

Reply:
> 2\. Emit an error message if `HOMEBREW_CURL` is set but is too old.

This seems like a good call to me 👍. It may be confusing to the user if it's set and silently not used. To be explicit: presumably you mean if `HOMEBREW_FORCE_BREWED_CURL` or `HOMEBREW_CURL_PATH` are passed and are too old (and the same deal for `git`).

Thanks @maxim-belkin!